sdk/scheduler/src/main/java/com/mesosphere/sdk/scheduler/plan/strategy/CanaryStrategy.java

```diff
@@ -42,8 +42,10 @@
 
     @Override
     public void proceed() {
+        super.proceed();
+        
         if (!initialized.get())  {
-            logger.warn("Proceed has no effect before strategy initialization.");
+            logger.warn("Proceed has no effect to children before strategy initialization.");
```

Comment:
I think the old wording was more intuitive -- I could see this causing users to ask "children of what?"